coordinator/crypto/crypto.go

```diff
@@ -44,13 +45,23 @@ func GenerateCert(
 		return nil, nil, fmt.Errorf("generating serial number: %w", err)
 	}
 
+	var additionalDNSNames []string
```

Comment:
We should have a unit test for this func. For now it would be ok if you only add cases that cover your change.